### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by SMBEN** [mtex-toolbox/mtex#1406] [Ask Anything] [answered]
Hi 
I want to change the scale in IPF map 
What exactly do I need to write in order to  change the Scale ??
thank you.

**Comment by lfgmorales** [comment]
Did you try

caxis ([min max])

After the colorbar line on your code?

Cheers

L




Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 20 Apr 2022, at 05:22, SMBEN ***@***.***> wrote:

﻿

Hi
I want to change the scale in IPF map
What exactly do I need to write in order to change the Scale ??
thank you.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1406>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN5VLZE6KSUVZUKWEMLVF5Z5JANCNFSM5T2VIJBQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1406", "url": "https://github.com/mtex-toolbox/mtex/discussions/1406", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by SMBEN** [reply]
![co](https://user-images.githubusercontent.com/99767925/164160889-044a88b0-663a-4fad-b980-fd94a4798e53.jpg)
![Mg Walid 2P](https://user-images.githubusercontent.com/99767925/164161085-f35c1023-50f4-4817-9e42-b673c3af0765.jpg)
This is my code 
what should I do ??

**Reply by SMBEN** [reply]
I want to have an IPF map with  scale 30µm, not 2µm.
How can I change it, please.

**Comment by lfgmorales** [comment]
﻿Would you mind to expand on “change the scale on IPF colormap”?

What exactly you want to do?



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 20 Apr 2022, at 08:04, SMBEN ***@***.***> wrote:

﻿

[co]<https://user-images.githubusercontent.com/99767925/164160889-044a88b0-663a-4fad-b980-fd94a4798e53.jpg>
[Mg Walid 2P]<https://user-images.githubusercontent.com/99767925/164161085-f35c1023-50f4-4817-9e42-b673c3af0765.jpg>
This my code
what should I do ??

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1406#discussioncomment-2598454>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6XRBQFGKFFS4SKIQ3VF6M4ZANCNFSM5T2VIJBQ>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1406#discussioncomment-2598454", "url": "https://github.com/mtex-toolbox/mtex/discussions/1406#discussioncomment-2598454", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by SMBEN** [reply]
Simply, I want to obtain an IPF map with scale 100 µm not to 2 µm like this.
How can I change it ??
![Mg Walid 2P](https://user-images.githubusercontent.com/99767925/164557365-23f72dae-5d1e-4e4d-95af-1edc8c5e8b4c.jpg)

**Comment by zmichels** [comment, marked as answer]
-->Hi there, Try something like the following (which sets the length to 10um)… but replace the first line with your plot command. 
```
plot(ebsd)
mP = mapPlot(gca)
mP.micronBar.length = 10 
```
Cheers,Zach

**Reply by SMBEN** [reply]
Hi, 
it does not work!

**Reply by SMBEN** [reply]
> -->Hi there, Try something like the following (which sets the length to 10um)… but replace the first line with your plot command.  ``` plot(ebsd) mP = mapPlot(gca) mP.micronBar.length = 10  ``` Cheers,Zach

Hi, It's work now.
Thank you very much.

**Comment by zmichels** [comment]
Sidenote… I think a 100um scalebar is going to be longer than the width of the plot you shared… which is why I suggested 10um. But try different values and see what works best. Cheers,Zach    From: SMBENSent: Thursday, April 21, 2022 2:47 PMTo: mtex-toolbox/mtexCc: SubscribedSubject: Re: [mtex-toolbox/mtex] help (Discussion #1406) Simply, I want to obtain an IPF map with scale 100 µm not to 2 µm like this.How can I change it ??—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
